src/ontology/config/robot_report_external_content.txt

```diff
@@ -0,0 +1,5 @@
+ERROR	file:tmp/mondo/src/sparql/qc/general/qc-syn-source-not-xref.sparql
+ERROR	file:tmp/mondo/src/sparql/qc/mondo/qc-animal-disease-rare.sparql
+ERROR	invalid_xref
+ERROR	duplicate_scoped_synonym
+ERROR	file:tmp/mondo/src/sparql/qc/general/qc-duplicate-exact-synonym-no-abbrev.sparql
```

Comment:
No, this file has to be manually curated by Trish - its on the pending items on this PR.

Reply:
Yes, I have an update I'll add in the next ~1 hr.

Reply:
Check for proxy merges added now.

Reply:
Oh, ok. This isn't an accidental instance of errors being piped into this file. This is actually a config file representing errors we are trying to account for / filter out of our templates, yes? I'm unfamiliar with these `--profile`s.

Should the file not be a `.tsv` instead of `.txt`?
It looks like it has 2 columns. I'm supposing these profile configs can't have column headers on them? If they did it'd be easier for a lay person to understand the file.